src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -727,6 +727,13 @@ protected DynamoModel(IStartConfiguration config)
             LibraryServices.MessageLogged += LogMessage;
             LibraryServices.LibraryLoaded += LibraryLoaded;
 
+            EngineController = new EngineController(
```

Comment:
at first glance this scares me - why do we need to create a new EngineController instance???

Reply:
Yes, I forgot to remove this, we do not want to create a new EngineController object. I have removed this now.